packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -401,6 +401,24 @@ class TextPainter {
     return value & 0xF800 == 0xD800;
   }
 
+  /// Returns the offset for the character after `offset`.
```

Comment:
I am not 100% fluent in the Unicode terminology and open to naming suggestions...

Having said that, this is one meaning the [Unicode glossary](https://www.unicode.org/glossary/#character) defines for the Word Character:
> (1) The smallest component of written language that has semantic value; refers to the abstract meaning and/or shape, rather than a specific shape (see also glyph), [...]

which seems very fitting here...